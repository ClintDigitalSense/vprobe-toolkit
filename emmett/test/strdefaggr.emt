VMM1Hz {
   str = sprintf("%s:%d", "barf", 12);
   agr[str] <- 3;
}
