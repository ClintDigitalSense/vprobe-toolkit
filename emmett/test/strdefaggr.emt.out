(version 0.2)
(defaggr agr 0 0)

 ;;;;;
(vprobe VMM1Hz ( do  ( setint str    (sprintf "%s:%d"  "barf"  12 ))( aggr  agr () () 3)))